android-gradle-plugin-aspectj/src/main/kotlin/com/archinamon/api/AspectJCompileTask.kt

```diff
@@ -132,31 +135,52 @@ internal open class AspectJCompileTask : AbstractCompile() {
         }
 
         private fun findCompiledAspectsInClasspath(task: AspectJCompileTask, aspectsFromJar: Collection<String>) {
-            val classpath: FileCollection = task.classpath
-            val aspects: MutableSet<File> = mutableSetOf()
-
-            classpath.forEach { file ->
-                if (aspectsFromJar.isNotEmpty() && DependencyFilter.isIncludeFilterMatched(file, aspectsFromJar)) {
-                    logJarAspectAdded(file)
-                    aspects shl file
+            val classpath = task.classpath
+            if (classpath != null && aspectsFromJar.isNotEmpty()) {
```

Comment:
if classpath is null it gets handled later at the compile task action. no need to log the empty unless you want to but maybe if we had different logging levels it might be okay but we dont.